src/pages/home/report/ReportActionCompose.js

```diff
@@ -195,6 +195,7 @@ class ReportActionCompose extends React.Component {
 
         this.shouldAutoFocus = !props.modal.isVisible && (this.shouldFocusInputOnScreenFocus || this.isEmptyChat()) && props.shouldShowComposeInput;
 
+        this.maxComposerLines = this.props.isSmallScreenWidth ? CONST.COMPOSER.MAX_LINES_SMALL_SCREEN : CONST.COMPOSER.MAX_LINES;
```

Comment:
It seems like computing this only once in the composer doesn't work because `isSmallScreenWidth` can change w/o this component being unmounted + remounted.

It also doesn't need to be a state variable. Let's just put this ternary in `render`?